Tests/SwiftProtobufTests/Test_Required.swift

```diff
@@ -167,7 +167,7 @@ class Test_Required: XCTestCase, PBTestHelpers {
     }
```

Comment:
We do (https://github.com/apple/swift-protobuf/blob/master/Sources/SwiftProtobuf/TextFormatTypeAdditions.swift#L22), it still throws, which have an issue to clean up.  So for the moment I believe the `debugDescription` is a hair easier to deal with.  Once it does't throw, we're probably good to use it in testing asserts.